### PR TITLE
Fail if an attribute is set both in environment and via secrets.

### LINKED ISSUE #28: Warn if value is overriden both in secrets and the environment
# Scenario:

A overwrite is done inside `environments/myenv.cfg`: 

```
[component:example]
myvar = True
```

as well as in secrets `secrets/myenv.cfg`:
```

[component:example]
myvar = False
```

Obviously the overwrites do conflict.

# Expectation
Batou should discover this scenario and warn the user

### PR DESCRIPTION
**Pull request by icemac**
Fixes #28.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,9 +4,14 @@
 2.3b2 (unreleased)
 ------------------
 
-- NetLoc objects are now comparable. (#88)
+- NetLoc objects are now comparable.
```
Comment (ctheune) [inline CHANGES.md:7]: The changelog seems confused :)

**Review by ctheune — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (sallner) [reply CHANGES.md:7]: Here some links were added in the next lines.

Comment (icemac) [reply CHANGES.md:7]: Having links to the tickets in the change log makes it easier to access the tickets. We also added the links here to the other issues of this release.

Comment (ctheune) [reply CHANGES.md:7]: Ah, I misread the diff and you just added the numbers ... fair enough.

**Review by ctheune — APPROVED** [review]

**Comment by ctheune** [comment]
I guess you need to rebase ...

**Comment by sallner** [comment]
Rebase applied.

**Comment by ctheune** [comment]
Thanks!

**Comment by ctheune** [comment]
No. I just forgot to click the second approval button ... -_-

**Pull request merged**